src/styles/main.scss

```
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import "./sass_style/block.scss";
```

why don't you add the import of variables there?
```suggestion
@import "./sass_style/variable.scss";
@import "./sass_style/block.scss";
```